packages/instrument-utils/src/guards/shouldUseStripeLinkByMinimumAmount.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { getCart } from '@bigcommerce/checkout/test-mocks';
+
+import shouldUseStripeLinkByMinimumAmount from './shouldUseStripeLinkByMinimumAmount';
+
+describe('shouldUseStripeLinkByMinimumAmount()', () => {
+    const cart = getCart();
+    const notSupportedCart = {
+        ...cart,
+        cartAmount: 0.4,
+    };
+
+    it('returns true if cart fullfill Stripe requirements', () => {
+        expect(shouldUseStripeLinkByMinimumAmount(cart)).toBe(true);
+    });
+
+    it('returns false if cart fullfill Stripe requirements', () => {
+        expect(shouldUseStripeLinkByMinimumAmount(notSupportedCart)).toBe(false);
+    });
```

Comment:
Could we make the condition more specific?
Now it appears that the same condition can have two outcomes.